packages/itwin/grouping-mapping-widget/src/test/QueryBuilder.testdata.ts

```diff
@@ -1261,6 +1261,53 @@ export const testCases: QueryBuilderTestData = {
         },
       ],
     },
+    {
+      name: "when property is a model, return a model query string",
+      expectedResult:
+        "SELECT BisCore.B.ECInstanceId, BisCore.B.ECClassId FROM BisCore.B JOIN mockModeledElement ON mockModeledElement.UserLabel = 'displayValueString' OR mockModeledElement.CodeValue = 'displayValueString' WHERE mockModeledElement.ECInstanceId = BisCore.Element.Model.id",
```

Comment:
This looks backwards. `mockModeledElement.ECInstanceId = BisCore.Element.Model.id` should be part of the `ON` clause, not `WHERE`, while the `UserLabel` and `CodeValue` checks should be in the `WHERE` clause.

Reply:
Flipped.